### DISCUSSION TITLE
REAIOSScheduler.h Not Found Error

### DISCUSSION THREAD

**Discussion by Raamzeez** [software-mansion/react-native-reanimated#3575] [Q&A] [answered]
When trying to create a production build of my react-native project (bare workflow) with expo, the build fails when running Fastlane. Specifically, when Fastlane is compiling react-native-reanimated. 

<img width="1257" alt="Screen Shot 2022-09-15 at 8 24 06 PM" src="https://user-images.githubusercontent.com/37488506/190530951-d9aceb27-30ce-4362-9e20-7ced7da79efd.png">

Upon further inspection of my node_modules folder, it appears that REAIOSScheduler.h does exist in the `node_modules/react_native_reanimated/ios/native` directory. Does anyone know why this error keeps occurring? I have done multiple things, including reverting code changes to a previous version of my application that used to work, deleting and reinstalling all libraries (including react-native-reanimated) numerous times, and clearing the cache. 

<img width="1438" alt="Screen Shot 2022-09-15 at 8 33 13 PM" src="https://user-images.githubusercontent.com/37488506/190531747-cff62c81-a98c-4c9a-97cf-af3ce40ffd77.png">

<img width="1440" alt="Screen Shot 2022-09-15 at 8 35 34 PM" src="https://user-images.githubusercontent.com/37488506/190531962-16691379-59df-424c-a281-99befdb0c818.png">

`"react-native-reanimated": "~2.3.1",` <-- As reported in package.json

**Comment by tomekzaw** [comment, marked as answer]
Hey, please don't use 2.3.1. The latest version is 2.10.0. Let us know if it works for you.

**Reply by Raamzeez** [reply]
Thanks for the help! I guess there was some issue with expo where it was restricting react-native-reanimated from getting upgraded even though I kept uninstalling and reinstalling it, and I didn't notice that it was not the latest version. This appears to have solved my issue. I am getting other issues from my project but the error doesn't seem to be coming from react-native-reanimated, so I am confident that this solved the issue.

**Reply by Raamzeez** [reply]
Quick Update: I was able to fix the other issues, which involved removing unused packages with yarn remove [package-name] in my project because there were linker issues happening with XCode.
